### PR TITLE
Added downgrade notice to release notes draft

### PR DESCRIPTION
**Pull request by nickchase**

### CODE REVIEW

File: releases/release-1.10/release-notes-draft.md [hunk 1]
```diff
@@ -50,6 +50,8 @@ In terms of networking, Kubernetes 1.10 is about control. Users now have beta su
 
 # Before Upgrading
 
+* If you need to downgrade from 1.10 to 1.9.x, downgrade to v1.9.6 to ensure PV and PVC objects can be deleted properly.
```
Comment (msau42) [inline releases/release-1.10/release-notes-draft.md]: Do we support multi-release downgrades?  ie, could the user downgrade from 1.10.x to 1.8.x?

**Comment by calebamiles** [comment]
I don't think there's any need to update this information here, @nickchase since this file has already become part of `CHANGELOG-1.10`

**Pull request closed without merging**